src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -361,6 +378,7 @@ export class ConnectionDialogService implements IConnectionDialogService {
 		if (this._model) {
 			this._model.dispose();
 		}
+		//Consider editing here?  We update model's provider here before showing dialog.  _configurationService available.
```

Comment:
typically the comments should have questions about what to do in the code.  Please rewrite with more context, or make the recommended updates.

Reply:
I was notating _possible_ places to make edits.   I can clean this up.  It probably just needs to removed.